### PR TITLE
T0404 mig reports

### PR DESCRIPTION
**Pull request by simgonzalez**

### CODE REVIEW

File: sponsorship_reporting/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "summary": "Add the reports for the sponsorships",
+    "version": "14.0.1.0.0",
+    "author": "Compassion CH",
+    "category": "Finance",
+    "website": "http://www.compassion.ch",
+    "depends": [
+        "account_fiscal_year",  # OCA/account-financial-reporting
+        "account_fiscal_month",  # OCA/account-financial-reporting
+        "sponsorship_sub_management",  # CompassionCH/compassion-modules
+        "crm_phone",
```
Comment (ecino) [inline sponsorship_reporting/__manifest__.py]: Is this really needed? Can we remove it?

Comment (simgonzalez) [reply sponsorship_reporting/__manifest__.py]: Yes they are needed, I already reduced the list of dependencies

Comment (ecino) [reply sponsorship_reporting/__manifest__.py]: crm_phone? can you let me know why?

Comment (simgonzalez) [reply sponsorship_reporting/__manifest__.py]: because of schedule_call method if my memory is good:D

Comment (ecino) [reply sponsorship_reporting/__manifest__.py]: I don't see any schedule_call in this code. Please point the reference or remove this dependency and check if the module is working without it.

Comment (simgonzalez) [reply sponsorship_reporting/__manifest__.py]: I tried without it and it seems to work as good.

**Pull request merged**